<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graf</class>
 <widget class="QWidget" name="Graf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(63, 199, 220);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>591</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget_graf">
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_result_graf_min_2">
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(0, 0, 0);
</string>
        </property>
        <property name="text">
         <string>Result</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_result_graf">
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(255, 255, 255);
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_check_graf">
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_exit_graf">
        <property name="styleSheet">
         <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_error_graf">
      <property name="styleSheet">
       <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_login">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;
color: rgb(255, 64, 64);
background-color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string> Login:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_graf_var">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
